src/plugins/irc-events/topic.js

```diff
@@ -9,11 +9,9 @@ module.exports = function(irc, network) {
 			return;
 		}
 		var from = data.nick || chan.name;
-		var self = false;
-		if (from.toLowerCase() == irc.me.toLowerCase()) {
-			self = true;
-		}
+		var self = (from.toLowerCase() == irc.me.toLowerCase());
```

Comment:
About the style: I wasn't trying to specifically change the style (or play any games...), but I think that was more about simplifying the code. I know I should have used '===', but either didn't notice it or think about it at the time and I was trying to keep the changes to a minimum. Even though this isn't strictly speaking on the scope of fixing the topic, I thought a small simplification wouldn't hurt anyone.